src/base/http/server.cpp

```diff
@@ -62,7 +62,7 @@ namespace
         {
             return std::none_of(badCiphers.cbegin(), badCiphers.cend(), [&cipher](const QString &badCipher)
             {
-                return cipher.name().contains(badCipher, Qt::CaseInsensitive);
+                return cipher.name().contains(badCipher, Qt::CaseInsensitive) || cipher.name().startsWith(u"dhe"_qs);
```

Comment:
I am away from my machine: Is this place to also filter out RSA key exchange ciphers?

Reply:
I am also away from my machine, I submitted this on mobile; but yes, this is where it would be filtered out.

As a side note, the reason I added that if block instead of adding DHE to badCiphers is that ECDHE ciphers would get caught up and blocked as well, which are the ciphers we want to use (https://www.ibm.com/support/pages/how-disable-ssltls-diffie-hellman-keys-less-2048-bits). That will prevent us from getting a lower rating on key exchange as well.